docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -1,80 +1,82 @@
 ---
 date:
   created: 2023-08-08
-pin: true
-links:
-    - External link: https://google.com
-
- 
+categories:
+    - Scraping
+    - Twitter
+    - Python
+authors:
+    - nrapstin
 ---
 
-
 # Using Twarc python package to scrape Twitter
 
-In this topic guide, we discuss how to get started with the [`twarc`](https://github.com/DocNow/twarc) python package to scrape Twitter.
+In this blog, we discuss how to get started with the [`twarc`](https://github.com/DocNow/twarc){:target="_blank"} python package to scrape Twitter.
+
+<!-- more -->
 
 ## Installing `Twarc` package
 We will use a `conda` to install all the python packages into. 
 That way we can make it into the kernel on JupyterHub and use the same environmnet on interactive yens, 
-[`yen-slurm`](/training/3_yen_slurm.html) or 
-[JupyterHub](/training/6_jupyter_hub.html).
+[`yen-slurm`](/_user_guide/slurm/) or 
+[JupyterHub](/_getting_started/jupyter/).
 
-```
-$ ml anaconda3
+```title="Terminal Command"
+ml anaconda3
```

Comment:
I don't think we should use conda anymore and I don't thing twarc works ...so idk how useful this info is

Reply:
included update card saying we recommend venv instead of conda